opendatahub-io/data-science-pipelines-operator
# Removed VisualizationServer and MLMD Writer

## The issue resolved by this Pull Request:
Resolves: https://github.com/opendatahub-io/data-science-pipelines-operator/issues/495

## Description of your changes:


This PR removes the VisualizationServer and MLMD Writer deployments for V2.

## Testing instructions


- Deploy https://github.com/opendatahub-io/data-science-pipelines-operator/blob/fa00b5830e49f7bd8b93d0f7ba7de6d920d25e3e/config/samples/dspa_simple.yaml. MLMD Writer should be deployed.
- Deploy https://github.com/opendatahub-io/data-science-pipelines-operator/blob/fa00b5830e49f7bd8b93d0f7ba7de6d920d25e3e/config/samples/dspa_simple_v2.yaml. MLMD Writer and VisualizationServer should not be deployed.

## Checklist
